trnsysGUI/diagram/Scene.py

```diff
@@ -159,6 +159,18 @@ def keyPressEvent(self, event):
             self.parent().moveDirectPorts = not self.parent().moveDirectPorts
             self.logger.debug("Changing move bool to " + str(self.parent().moveDirectPorts))
 
+        if event.key() == Qt.Key_Delete:
+            self.logger.debug("Delete detected")
+            for c in self.parent().trnsysObj:
+                # Delete connection
```

Comment:
Can you remove the debug statement: it's not really useful in a log file only maybe for "debugging" during development. ;)